File: python/selection_sort.py
def sort(arr: list[int]) -> list[int]:
    min = 0


    for i in range(0, 9):
        min = i
        for j in range(i+1, 10):
            if arr[j] < arr[min]:
                min=j

        arr[min], arr[i] = arr[i], arr[min]

    return arr

File: src/shah/jeevan/SelectionSort.java
package shah.jeevan;

import java.util.Arrays;

public class SelectionSort {
    public static void sort(int[] arr) {
        int min;
        int temp;

        for (int i = 0; i < arr.length; i++) {
            min = i;
            for (int j=i+1; j < arr.length; j++) {
                if (arr[j] < arr[min]) {
                    min = j;
                }
            }
            temp = arr[min];
            arr[min] = arr[i];
            arr[i] = temp;
        }
        System.out.println(Arrays.toString(arr));
    }
}
